.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,19 @@
+<!-- Hey there. Thank you so much for improving our app. Please take a moment to fill out the following suggestion on how to structure this PR description. Having roughly the same layout drastically helps everyoby :)-->
+
+#### long description of the changes in your PR
+*Now you can make videos*
+
+#### Fixes the following issue(s)
+<!--Also add reddit or other links where you change is related-->
+- #1234
```

Comment:
Just asking, will it quote issue 1234? I don't think it's a great idea, just say paste link here, if they are github lins it will format greatly.

Reply:
okay, sure. Thats just how I saw it in most issues and think its the cleanest way of doing it